src/doc/reference.md

```diff
@@ -2070,6 +2070,9 @@ macro scope.
    trait of the same name. `{Self}` will be replaced with the type that is supposed
    to implement the trait but doesn't. To use this, the `on_unimplemented` feature gate
    must be enabled.
+- `must_use` - on structs and enums, will warn if this value isn't bound to a variable.
```

Comment:
But isn't `some_fn(my_var)` a use of `my_var`, too? This doesn't warn, however. What *does* actually warn is an expression statement `<expr>;` where `<expr>` is a `#[must_use]` type.